environments/nodejs/package.json

```diff
@@ -14,13 +18,10 @@
   },
   "dependencies": {
     "body-parser": "*",
-    "co": "~4.6.0",
```

Comment:
This is confusing... I see that you added a commit for adding these dependencies back, but the final diff still shows the dependencies removed.  Maybe some merge error?

Reply:
My bad, the file updated from the commit was `examples/nodejs/package.json`, I forgot to update `environments/nodejs/package.json` as well. 

Please check the latest commit. Sorry